### ISSUE TITLE
Installing with Docker now working

### ISSUE THREAD

**Issue by adapik** [prooph/proophessor-do-symfony#16]
I'm following instructions. EventStore and tables were created but seems projections are not. 

`An exception occurred while executing 'SELECT * FROM read_user':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todo.read_user' doesn't exist`

**Comment by prolic** [comment]
You have to run the projections, did you?

**Comment by adapik** [comment]
Yes, I've finally found commands for running projections in manual installation file and run them mannualy. Now it's working. They are absent in Docker installation instructons.

**Comment by prolic** [comment]
Really? Please provide a PR for the docs. Thanks.

**Comment by codeliner** [comment]
projections should start automatically when using docker: https://github.com/prooph/proophessor-do-symfony/blob/master/docker-compose.yml#L37

But maybe there is a timing problem when running docker-compose up -d for the first time :thinking: 
I'll check it again with a fresh installation.

**Comment by adapik** [comment]
Thanks a lot for your help!

@codeliner , you are right. After docker-compose down and up everything works properly.  I suppose when I was trying to run docker-compose up for the first time it was some problem with db (not started, not configured?), but it was no exit code 1 so i didn't noticed.

Probably it's some my local problem and you should close the issue.

**Comment by codeliner** [comment]
Ok, cool. Good to know. We should add a hint in the docs that if projections are not running one should look at the containers and try a down and up again
